constitution.md

```diff
@@ -4,11 +4,12 @@
 
 The SDSM&T Unmanned Aerial Vehicle Team will be one that designs
 aerial robots with the intent of taking part in various
-competitions. The purpose of the team is to promote aerial robotics in
-the SDSM&T community and to help students learn valuable skills
-associated with building aerial robots. As a byproduct of taking part
-in competitions, SDSM&T will benefit by becoming more familiar to
-people in the unmanned aerial robotics community.
+competitions, in particular the International Aerial Robotics Competition (IARC). 

```

Comment:
Change this to "taking part in the International Aerial Robotics Competition".  Other competitions are the purview of other teams.